docs/source/documentation.html.haml

```diff
@@ -0,0 +1,130 @@
+---
+title: Algolia Places - Documentation
+---
+%div.navigation
+  %div.fl-left
+  %a{href: "/index.html"}
+    %span.brand
+      <span class="icon icon-algolia-light">Algolia</span> Community
+    %span.pipe
+    %span Places
+
+
+  %div.fl-right
+    %div.searchbar
+      %input{class: "form-control aa-input", placeholder: "Search in the documentation", type: "text", id: "doc-search"}
+
+    %ul
+      %li.active
+        %a{href: "/documentation.html"} documentation
+      %li
+        %a examples
+      %li
+        %a about
+      %li
+        %a
+          %span.icon.icon-github
+            github
+
+
+%section.hero-section.shrink
+  .container
+    %div.hero-container
+
+      %div.fl-left
+        %span{class: "icon icon-algolia-small", id: "icon-algolia"}
+        %span{class: "no-desktop small-illustration"}
+          %img{src: "images/svg/places-illustration.svg"}
+        %h1 Documentation
+        %h3 The easiest way to find an address lorem dolor sit amet en consequeter
+
+
+%section.documentation-section
+  .container
+    %nav.sidebar#sidebar
+      %ul
+        %li
+          %a{href: "#introduction"} Introduction
+
+        %li
+          %a{href: "#setup"} Setup
+
+        %li
+          %a{href: "#options"} Options
+
+    %div.documentation-container
+      %article{id: "introduction"}
+        %h2 Introduction
+
+        %p
+          We’re scratching our own itch here. As developers, we spend a lot of time
+          reading documentation, and it isn’t always easy to find the information we
+          need.
+
+        %p
+          Not blaming anyone here. Building a good search for a documentation is a
+          complex challenge. We happen to have a lot of experience doing that, and we
+          want to share it with the world. For free.
+
+        %p
+          Just submit the form on the website and we’ll get back to you with what you
+          need to integrate your new search into your website.
+        %ul
+          %li We’ll crawl your documentation pages, 
+          %li We’ll configure your search experience, 
+          %li You’ll need to add a bit of JavaScript and CSS code to your website.
+
+
+      %article{id: "setup"}
+        %h2 Setup
+
+        %p
+          Once we’ve crawled your documentation website we’ll send you the credentials you need to add the following code snippet to your website:
+
+        :code
+          # lang: html
+          <link rel="stylesheet" href="//cdn.jsdelivr.net/docsearch.js/1/docsearch.min.css" /> 
+          <script type="text/javascript" src="//cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js"></script>
+          <script type="text/javascript">
+            docsearch({ 
```

Comment:
You have weird line terminators, see 
![2016-03-29-181037_947x167_scrot](https://cloud.githubusercontent.com/assets/123822/14115003/94a7cb0a-f5d9-11e5-916f-14d3a7649f83.png)

Try to remove them. You can also update your programming font to maybe spot them. (Inconsolata may show them)